examples/utils/example-utils/src/modelLoader/modelContainerRuntimeFactory.ts

```diff
@@ -37,15 +37,17 @@ export abstract class ModelContainerRuntimeFactory<ModelType> implements IRuntim
 		context: IContainerContext,
 		existing: boolean,
 	): Promise<IRuntime> {
-		const runtime = await ContainerRuntime.load(
+		const runtime = await ContainerRuntime.loadRuntime({
 			context,
-			this.registryEntries,
+			registryEntries: this.registryEntries,
 			// eslint-disable-next-line import/no-deprecated
-			makeModelRequestHandler(this.createModel.bind(this)),
-			this.runtimeOptions,
-			undefined, // scope
+			requestHandler: makeModelRequestHandler(this.createModel.bind(this)),
+			provideEntryPoint: () => {
+				throw new Error("TODO: AB#4990");
```

Comment:
If we can't address this in this PR, let's reword this error message.  AB# can go in a comment.  Same elsewhere